gpbft/participant_test.go

```diff
@@ -84,12 +84,13 @@ func (pt *participantTestSubject) expectBeginInstance() {
 	// Prepare the test host.
 	pt.host.On("GetCanonicalChain").Return(pt.canonicalChain, *pt.powerTable, pt.beacon)
 	pt.host.On("Time").Return(pt.time)
-	pt.host.On("NetworkName").Return(pt.networkName)
+	pt.host.On("NetworkName").Return(pt.networkName).Maybe()
+	pt.host.On("MarshalPayloadForSigning", mock.AnythingOfType("*gpbft.Payload")).
+		Return([]byte(gpbft.DOMAIN_SEPARATION_TAG + ":" + pt.networkName))
 
```

Comment:
Is this the right way to do this? Basically:

1. We may _not_ call `NetworkName` now, because that's handled inside of `Host.MarshalPayloadForSigning`.
2. Whenever we call `Sign`, we call `Host.MarshalPayloadForSigning`.